Manifest.toml

```diff
@@ -0,0 +1,236 @@
+[[Arpack]]
```

Comment:
Isn't it the right way to handle dependencies with Pkg 3? I thought we could still use REQUIRE that creates a Project.toml file but I also thought we should use the new environments provided by Pkg3. Odow uses it in Kokako.jl for example. Am I wrong?

Reply:
I add a `Manifest.toml` because it allows me to depend upon a specific JuMP commit when testing on Travis. If you just want the latest tagged version of each package, `Project.toml` is sufficient.

Reply:
Thanks!

Reply:
So let's keep `Project.toml` then!